zircon.core/src/jvmTest/kotlin/org/hexworks/zircon/CharacterTilesAsStringTest.kt

```diff
@@ -0,0 +1,101 @@
+package org.hexworks.zircon
+
+import org.assertj.core.api.Assertions.assertThat
+import org.hexworks.zircon.api.data.Position
+import org.hexworks.zircon.api.data.Size
+import org.hexworks.zircon.api.data.Tile
+import org.hexworks.zircon.internal.graphics.DefaultTileComposite
+import org.junit.Test
+
+class CharacterTilesAsStringTest {
```

Comment:
Is it possible to make this a `commonTest` test?

Reply:
I'll look into it.

Reply:
Looks like yes, I just have to make peace with not using AssertJ :cry: